lib/dash/dash_parser.js

```diff
@@ -212,6 +212,30 @@ shaka.dash.DashParser = class {
     // No-op
   }
 
+  /**
+   * @override
+   * @exportInterface
+   */
+  onInitialVariantChosen(variant) {
+    if (this.manifest_ && this.lowLatencyMode_ &&
+        this.minTotalAvailabilityTimeOffset_) {
+      const stream = variant.video || variant.audio;
+      if (stream && stream.segmentIndex) {
+        const nowSeconds = Date.now() / 1000.0;
+        const position = stream.segmentIndex.find(nowSeconds);
```

Comment:
It looks like you're assuming here that the presentation timeline is always aligned with wall-clock time.  This is not generally true.

The docs for find():

> Finds the position of the segment for the given time, in seconds, relative to the start of the presentation.

This should be a timestamp in presentation time, not wall-clock time.  I think you want the time of the live edge as computed by `manifest_.presentationTimeline.getSegmentAvailabilityEnd()`.

Reply:
Changed!